### ISSUE TITLE
Add alternative IMFs

### ISSUE THREAD

**Issue by kadrlica** [DarkEnergySurvey/ugali#15]
Currently we only have Chabrier, we should at least add Kroupa.

**Comment by kadrlica** [comment]
@sazabi4 once mentioned some interest in this.

**Comment by sazabi4** [comment]
I think it's both IMF and isochrone. We should add in an empirical isochrone as well as IMF to 

1) make the isochrone best representative to the dwarf galaxies in the RGB region. @kadrlica has sort of started this a while ago. I found a plot I made a long time ago and may help some of the work on this.
![gruii_w_dwarf_members](https://cloud.githubusercontent.com/assets/7751063/25050911/9c51cd80-210f-11e7-906b-efbb4c004aff.png)

Probably this first one should go to a different issue thread.

2) make the isochrone/IMF go brighter than Mv = -2. I think the TRGB should go up to Mv = -4 but I think the padova isochrone goes to Mv = -3 but somehow the Chabrier IMF doesn't give anything at Mv = -2.5 or brighter. I'm not sure if that's true and I think further investigation is required. 

Just in case you are interested, we brought up the 2nd discussion at the LSST Hack. This question was brought up as it affects the understanding of the furthest dwarf galaxy in Local Group that can have resolved stars observed by LSST.
